addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1880,9 +1880,9 @@ export class Order extends PosModel {
         }
     }
 
-    initialize_validation_date() {
-        this.validation_date = new Date();
-        this.formatted_validation_date = formatDateTime(DateTime.fromJSDate(this.validation_date));
+    initialize_date_order() {
+        this.date_order = luxon.DateTime.now();
+        this.formatted_date_order = formatDateTime(this.date_order);
```

Comment:
can we remove this and use the formatted version of the `date_order` whenever we need it rather than having the attribute?
We need the string formatted date in the receipt print I guess.

Reply:
Hello @paj-odoo 
Suggested changes are done.
Thanks!